incubator/yarn-plugin-external-workspaces/.gitignore

```
@@ -0,0 +1 @@
+!dist/*.cjs
```

Nit: Missing newline at end of file:
```suggestion
!dist/*.cjs

```